Mathlib/Combinatorics/HalesJewett.lean

```diff
@@ -78,6 +169,7 @@ Formally, a line is represented by a word `l.idxFun : ι → Option α` which sa
 `l.idxFun i = some y`).
 
 When `α` has size `1` there can be many elements of `Line α ι` defining the same function. -/
+@[ext]
 structure Line (α ι : Type*) where
```

Comment:
Is a `Line` a `Subspace`. (And the predicate `nontrivial` would be meaningful here.)